cmd/ethatomicswap/contract/src/contracts/AtomicSwap.sol

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2017 Altcoin Exchange, Inc
+// Copyright (c) 2018 The Decred developers and Contributors
```

Comment:
This shouldn't have a Decred copyright, it's not our code.  You're welcome to add your own personal copyright if this contains your changes.

Reply:
Sounds fair, I'll replace decred with my organisation than. You are correct.